packages/sdk/src/resolve/resolve-accounts.js

```diff
@@ -35,8 +36,9 @@ async function collectAccounts(ix, accounts, last, depth = 3) {
     if (Array.isArray(ax)) {
       await collectAccounts(ix, ax, old, depth - 1)
     } else {
+      if (ax.addr) ax.addr = sansPrefix(ax.addr)
```

Comment:
Make all addresses sansPrefix to remove the authz function sansPrefix requirement